Box.V2/Models/BoxZipDownloadStatus.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Box.V2.Models.Request
+{
+    /// <summary>
+    /// Box representation of the status of a download for a zip file
+    /// </summary>
+    public class BoxZipDownloadStatus
+    {
+        public const string FieldTotalFileCount = "total_file_count";
+        public const string FieldDownloadedFileCount = "downloaded_file_count";
+        public const string FieldSkippedFileCount = "skipped_file_count";
+        public const string FieldSkippedFolderCount = "skipped_folder_count";
+        public const string FieldState = "state";
+
+        /// <summary>
+        /// The total number of files of the zip 
+        /// </summary>
+        [JsonProperty(PropertyName = FieldTotalFileCount)]
+        public int TotalFileCount { get; set; }
+
+        /// <summary>
+        /// The number of files in the zip that were downloaded
+        /// </summary>
+        [JsonProperty(PropertyName = FieldDownloadedFileCount)]
+        public int DownloadedlFileCount { get; set; }
+
+        /// <summary>
+        /// The number of files in the zip that were skipped when downloading
+        /// </summary>
+        [JsonProperty(PropertyName = FieldSkippedFileCount)]
+        public int SkippedFileCount { get; set; }
+
+        /// <summary>
+        /// The number of folders in the zip that were skipped when downloading
+        /// </summary>
+        [JsonProperty(PropertyName = FieldSkippedFolderCount)]
+        public int SkippedFolderCount { get; set; }
+
+        /// <summary>
+        /// The state of the download of the zip file
+        /// </summary>
+        [JsonProperty(PropertyName = FieldState)]
+        [JsonConverter(typeof(StringEnumConverter))]
+        public BoxZipDownloadState State { get; set; }
+
+        /// <summary>
+        /// A list of naming conflicts among the files and folders in the zip. This is manually appended in the BoxFilesManager.DownloadZip() method.
+        /// </summary>
+        [JsonProperty(PropertyName = "none")]
+        public List<List<BoxZipConflictItem>> NameConflicts { get; set; }
```

Comment:
Move to BoxZipInfo to match the [Box Java SDK](https://github.com/box/box-java-sdk/blob/master/src/main/java/com/box/sdk/BoxZipInfo.java#L73).

Reply:
Also, need to create a BoxZipConflict class to match Box Java SDK.